internal/corazawaf/rule.go

```diff
@@ -295,7 +307,7 @@ func (r *Rule) doEvaluate(phase types.RulePhase, tx *Transaction, cache map[tran
 
 		}
 		if r.ID_ != 0 {
-			// we avoid matching chains and secmarkers
+			// we avoid matching secmarkers
```

Comment:
Not related to the PR but noticed while trying to grok the function. We already checked `ParentID_` above so I think this `ID_` check is only to handle the secmarker case, not chain case